packages/UsageAnalysis/files/Test Track/Queries/Deleting.md

```diff
@@ -1,8 +1,8 @@
-1. Go to **Browse** > **Databases** and find queries from the previous steps (3.1)
-1. Delete them one after the other:
+1. Go to **Browse** > **Databases** > **PostgresDart** > **NorthwindTest** and find query from the previous steps with name `new_test_query`.
+1. Delete it:
```

Comment:
Don't, use 'it,' 'this', and so on; reference to the exact object by name instead.

Reply:
changed